test/functional/test_node_ports.py

```diff
@@ -111,16 +112,24 @@ def test_start_node1_then_start_node2_no_conflict_ports(self):
                                   subprocess.PIPE, subprocess.PIPE)[1]
         qemu_result = run_command(PS_QEMU, True,
                                   subprocess.PIPE, subprocess.PIPE)[1]
+        if 'dell' in self.node_info['type']:
+            racadm_result = run_command(PS_RACADM, True,
+                                        subprocess.PIPE, subprocess.PIPE)[1]
+
         # check if node test1 is running
         assert "test1" in socat_result
         assert "test1" in ipmi_result
         assert "test1-node" in qemu_result
+        if 'dell' in self.node_info['type']:
+            assert "test1" in racadm_result
 
         self.node_info_2['name'] = "test2"
         # modify node configuration to resolve port conflict
         self.node_info_2['ipmi_console_ssh'] = 9301
         self.node_info_2['ipmi_console_port'] = 9001
         self.node_info_2['bmc_connection_port'] = 9101
+        if 'dell' in self.node_info_2['type']:
+            self.node_info_2['racadm_port'] = 10023
```

Comment:
As comment above, the code here should be:

    self.node_info_2["racadm"] = {}
    self.node_info_2["racadm"]["port"] = 10023

Below 4 tests need updates as well.